pkg/cli/tsdump.go

```
@@ -194,6 +192,10 @@ will then convert it to the --format requested in the current invocation.
 		if convertFile == "" {
 			// To enable conversion without a running cluster, we want to skip
 			// connecting to the server when converting an existing tsdump.
+			if cliCtx.clientOpts.User != username.RootUser {
+				serverCfg.User, _ = username.MakeSQLUsernameFromUserInput(cliCtx.clientOpts.User, username.PurposeValidation)
```

nit: just add a comment on why we are ignoring the error?